src/languageclient/WrappedErrorHandler.ts

```diff
@@ -24,37 +31,30 @@ export class WrappedErrorHandler implements ErrorHandler {
      *  be reset if a message got successfully send or received.
      */
     public error(error: Error, message: Message | undefined, count: number): ErrorAction {
-        let parsed = parseError(error);
-        ext.reporter.sendTelemetryEvent(
-            languageServerErrorTelemId,
-            <TelemetryProperties>{
-                error: parsed.errorType,
-                errorMessage: parsed.message,
-                result: "Failed",
-                jsonrpcMessage: message ? message.jsonrpc : "",
-                count: String(count),
-                stack: parsed.stack
-            },
-            {
-                secondsSinceStart: (Date.now() - serverStartMs) / 1000
-            });
+        // Use our shared error handling code to notification telemetry and user of the error
+        // in a standard way
+        callWithTelemetryAndErrorHandlingSync(languageServerErrorTelemId, (context: IActionContext) => {
+            context.telemetry.properties.jsonrpcMessage = message ? message.jsonrpc : "";
+            context.telemetry.measurements.secondsSinceStart = (Date.now() - this._serverStartTime) / 1000;
```

Comment:
Does it make sense to use stopwatch here? I see it being used in the other PR for the same purpose.

Reply:
Perhaps, but it's a convenience class that really does add much, so I'll leave it for now.  Thanks!